v1/app.py

```diff
@@ -56,13 +56,14 @@ def login():
     user_email = request.get_json(force=True)['email']
     user_password = request.get_json(force=True)['password']
     user_admin = request.get_json(force=True)['admin']
+    username = request.get_json(force=True)['username']
```

Comment:
Consider optimizing this functionality

`user_data = request.get_json(force=True)`
`username = user_date['username']` or `username = user_data.get('username')`